apps/glusterfs/app_cluster.go

```diff
@@ -150,6 +150,7 @@ func (a *App) ClusterInfo(w http.ResponseWriter, r *http.Request) {
 		if err != nil {
 			return err
 		}
+		Lookup(tx).CompleteClusterInfo(info)
```

Comment:
Even if it's currently just returning nil, we should still treat a possible future error condition here.